plugin/pom.xml

```diff
@@ -46,6 +46,33 @@
   </developers>
 
   <dependencies>
+
+    <dependency>
+      <groupId>org.everit.json</groupId>
+      <artifactId>org.everit.json.schema</artifactId>
+      <version>1.3.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>vault</artifactId>
+      <version>1.10.6</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.5</version>
```

Comment:
@oleg-nenashev does Jenkins not already come with a JSON module?

Reply:
Doesn’t it use org.json?